### ISSUE TITLE
Variable classifier_dropout

### ISSUE THREAD

**Issue by KoichiYasuoka** [WhereIsAI/BiLLM#3]
I've just found that `classifier_dropout` is fixed to 0.1 in both `LlamaForTokenClassification` and `MistralForTokenClassification` as:

```
        self.dropout = nn.Dropout(0.1)
```

however in `GPT2ForTokenClassification` of HuggingFace Transformers it is enhanced as:

```
        if hasattr(config, "classifier_dropout") and config.classifier_dropout is not None:
            classifier_dropout = config.classifier_dropout
        elif hasattr(config, "hidden_dropout") and config.hidden_dropout is not None:
            classifier_dropout = config.hidden_dropout
        else:
            classifier_dropout = 0.1
        self.dropout = nn.Dropout(classifier_dropout)
```

and now we are trying to include `LlamaForTokenClassification` and `MistralForTokenClassification` into HuggingFace Transformers at https://github.com/huggingface/transformers/pull/29878 . Please show us better way to include them.

**Comment by SeanLee97** [comment]
@KoichiYasuoka Thank you for your suggestions! I will add a feature to support classifier_dropout.

B.T.W, the BiLLM's implementation for TokenClassification differs from the official one. In BiLLM, we convert the attention mask from uni- to bi-directional. This change can improve the performance of token classification significantly, according to our experiments in the paper https://arxiv.org/abs/2310.01208

**Comment by SeanLee97** [comment]
I've added a feature to support `classifier_dropout` in PR: https://github.com/WhereIsAI/BiLLM/pull/4

Now, you can specify `classifier_dropout` in billm>=0.1.2

**Comment by KoichiYasuoka** [comment]
Thank you @SeanLee97 and now I close this issue. See you later at https://github.com/huggingface/transformers/pull/29878 and https://github.com/huggingface/transformers/issues/29940
